Looks like Travis still reports a couple of style violations:

```
$ flake8 homeassistant --exclude bower_components,external
homeassistant/components/notify/nma.py:48:9: F401 'Request' imported but unused
homeassistant/components/notify/nma.py:59:40: E203 whitespace before ':'
homeassistant/components/notify/nma.py:75:9: F401 'Request' imported but unused
homeassistant/components/notify/nma.py:78:31: E203 whitespace before ':'
The command "flake8 homeassistant --exclude bower_components,external" exited with 1.

$ pylint homeassistant
************* Module homeassistant.components.notify.nma
W: 64,30: Use % formatting in logging functions but pass the % parameters as arguments (logging-format-interpolation)
W: 98,34: Use % formatting in logging functions but pass the % parameters as arguments (logging-format-interpolation)
```

--- a/homeassistant/components/notify/nma.py
+++ b/homeassistant/components/notify/nma.py
@@ -45,7 +45,7 @@ def get_service(hass, config):
 
     try:
         # pylint: disable=unused-variable
-        from requests import Request, Session
+        from requests import Session
 
     except ImportError:
         _LOGGER.exception(
@@ -56,12 +56,13 @@ def get_service(hass, config):
 
     nma = Session()
     response = nma.get(_RESOURCE + 'verify',
-                       params={"apikey" : config[DOMAIN][CONF_API_KEY]})
+                       params={"apikey": config[DOMAIN][CONF_API_KEY]})
     tree = ET.fromstring(response.content)
+    # pylint: disable=logging-not-lazy
     if tree[0].tag == 'error':
         _LOGGER.error(
             "Wrong API key supplied. "
-            "Error: {}".format(tree[0].text))
+            "{}".format(tree[0].text))
     else:
         return NmaNotificationService(config[DOMAIN][CONF_API_KEY])
 
@@ -72,10 +73,10 @@ class NmaNotificationService(BaseNotificationService):
 
     def __init__(self, api_key):
         # pylint: disable=no-name-in-module, unused-variable
-        from requests import Request, Session
+        from requests import Session
 
         self._api_key = api_key
-        self._data = {"apikey" : self._api_key}
+        self._data = {"apikey": self._api_key}
 
         self.nma = Session()
 
@@ -92,6 +93,7 @@ class NmaNotificationService(BaseNotificationService):
         response = self.nma.get(_RESOURCE + 'notify',
                                 params=self._data)
         tree = ET.fromstring(response.content)
+        # pylint: disable=logging-not-lazy
         if tree[0].tag == 'error':
             _LOGGER.exception(
                 "Unable to perform request. "

--- a/homeassistant/components/notify/nma.py
+++ b/homeassistant/components/notify/nma.py
@@ -58,11 +58,9 @@ def get_service(hass, config):
     response = nma.get(_RESOURCE + 'verify',
                        params={"apikey": config[DOMAIN][CONF_API_KEY]})
     tree = ET.fromstring(response.content)
-    # pylint: disable=bad-option-value
+
     if tree[0].tag == 'error':
-        _LOGGER.error(
-            "Wrong API key supplied. "
-            "{}".format(tree[0].text))
+        _LOGGER.error("Wrong API key supplied. %s", tree[0].text)
     else:
         return NmaNotificationService(config[DOMAIN][CONF_API_KEY])
 
@@ -93,8 +91,7 @@ class NmaNotificationService(BaseNotificationService):
         response = self.nma.get(_RESOURCE + 'notify',
                                 params=self._data)
         tree = ET.fromstring(response.content)
-        # pylint: disable=bad-option-value
+
         if tree[0].tag == 'error':
             _LOGGER.exception(
-                "Unable to perform request. "
-                "Error: {}".format(tree[0].text))
+                "Unable to perform request. Error: %s", tree[0].text)

--- a/homeassistant/components/notify/nma.py
+++ b/homeassistant/components/notify/nma.py
@@ -58,7 +58,7 @@ def get_service(hass, config):
     response = nma.get(_RESOURCE + 'verify',
                        params={"apikey": config[DOMAIN][CONF_API_KEY]})
     tree = ET.fromstring(response.content)
-    # pylint: disable=logging-not-lazy
+    # pylint: disable=bad-option-value
     if tree[0].tag == 'error':
         _LOGGER.error(
             "Wrong API key supplied. "
@@ -93,7 +93,7 @@ class NmaNotificationService(BaseNotificationService):
         response = self.nma.get(_RESOURCE + 'notify',
                                 params=self._data)
         tree = ET.fromstring(response.content)
-        # pylint: disable=logging-not-lazy
+        # pylint: disable=bad-option-value
         if tree[0].tag == 'error':
             _LOGGER.exception(
                 "Unable to perform request. "

